judge/models/problem.py

```diff
@@ -124,6 +124,10 @@ class Problem(models.Model):
     authors = models.ManyToManyField(Profile, verbose_name=_('creators'), blank=True, related_name='authored_problems',
                                      help_text=_('These users will be able to edit the problem, '
                                                  'and be listed as authors.'))
+    authorship_name = models.CharField(max_length=100, verbose_name=_("Author's name"), blank=True,
```

Comment:
Verbose name should be lower case. Django will capitalize as needed.
```suggestion
    authorship_name = models.CharField(max_length=100, verbose_name=_("author's name"), blank=True,
```

Reply:
Done :)